changelogs/fragments/33-vmware_guest_info_list_fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Handle list items in vSphere schema while handling facts in vmware_guest_info (https://github.com/ansible-collections/vmware/issues/33).
```

Comment:
I find the description a bit confusing. `_extract()` is used by `to_json()`. And so, potentially, the change can impact `vmware_datastore_info` and `vmware_host_auto_start` too.